code/modules/mob/living/simple_animal/hostile/vannatusk_ghostrole.dm

```diff
@@ -0,0 +1,76 @@
+/mob/living/simple_animal/hostile/vannatusk_ghostrole
```

Comment:
make this a subtype to cut down on copypaste.

Reply:
Is there any easy way to do this while also having the subtype ignore the code that makes them shoot spikes? I tried using istype to make it possible but I couldn't get it to work properly.

Reply:
override the procs.